examples/multiple_windows/lib/app/main_window.dart

```diff
@@ -0,0 +1,255 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// ignore_for_file: invalid_use_of_internal_member
+// ignore_for_file: implementation_imports
+
+import 'package:flutter/material.dart';
+import 'package:flutter/src/widgets/_window.dart';
+
+import 'window_controller_render.dart';
+
+import 'regular_window_content.dart';
+import 'window_settings_dialog.dart';
+import 'models.dart';
+import 'regular_window_edit_dialog.dart';
+
+class MainWindow extends StatelessWidget {
+  MainWindow({super.key, required BaseWindowController mainController}) {
+    windowManagerModel.add(
+      KeyedWindow(
+        isMainWindow: true,
+        key: UniqueKey(),
+        controller: mainController,
+      ),
+    );
+  }
+
+  final WindowManagerModel windowManagerModel = WindowManagerModel();
+  final WindowSettings settings = WindowSettings();
+
+  @override
+  Widget build(BuildContext context) {
+    return ViewAnchor(
+      view: ListenableBuilder(
+        listenable: windowManagerModel,
+        builder: (BuildContext context, Widget? _) {
+          final List<Widget> childViews = <Widget>[];
+          for (final KeyedWindow window in windowManagerModel.windows) {
+            if (window.parent == null && !window.isMainWindow) {
+              childViews.add(
+                WindowControllerRender(
```

Comment:
Thinking about efficiency here. Let's say that windowManagerModel has a new window added to it. This builder will rebuild, and a brand new WindowControllerRender will be created for every window, even the old ones. Any performance concern there?

I keep thinking hard about best practices in this example because I know tons of people are going to copy it.